public/javascripts/tab-forms.js

```diff
@@ -0,0 +1,17 @@
+function tabForm(evt, formName) {
+
+    var i, tabcontent, tablinks;
+
+    tabcontent = document.getElementsByClassName("tabcontent");
+    for (i = 0; i < tabcontent.length; i++) {
+        tabcontent[i].style.display = "none";
+    }
+
+    tablinks = document.getElementsByClassName("tablinks");
+    for (i = 0; i < tablinks.length; i++) {
+        tablinks[i].className = tablinks[i].className.replace(" active", "");
```

Comment:
Nice use of `className`!

It's definitely nicer to write `.classList.toggle('active')` instead, but... Manually replacing something in a string is way better than trying to work around [compatibility issues](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Browser_compatibility) of `classList`.